magento-research/pwa-studio
# [doc] Rewrite Peregrine Overview page

## Description

Rewrite the Peregrine Overview page to reflect current vision of the Peregrine library.

## Related Issue

Closes #1340 

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Lint the modified page: ` npm run lint src/peregrine/index.md`
3. Verify no linting errors associated with the page
4. Build the HTML preview server: `npm run develop`
5. Verify the site is successfully built
6. Navigate to `/peregrine`
7. Verify content is updated and different from currently published site.

## Screenshots / Screen Captures (if appropriate)

n/a

## Checklist:
